libs/@guardian/react-crossword/src/hooks/useUpdateCell.ts

```diff
@@ -0,0 +1,48 @@
+import { isUndefined } from '@guardian/libs';
+import { useCallback } from 'react';
+import type { Coords } from '../@types/crossword';
+import { useData } from '../context/Data';
+import { useProgress } from '../context/Progress';
+import { useValidAnswers } from '../context/ValidAnswers';
+
+export const useUpdateCell = () => {
+	const { setProgress } = useProgress();
+	const { setValidAnswers } = useValidAnswers();
+	const { cells } = useData();
+
+	const updateCell = useCallback(
```

Comment:
why the renaming from `setCellProgress`?

Reply:
Because it also resets the valid answers now